lib/lightning_web/components/new_inputs.ex

```diff
@@ -356,13 +361,119 @@ defmodule LightningWeb.Components.NewInputs do
     """
   end
 
+  def input(%{type: "toggle"} = assigns) do
+    assigns =
+      assigns
+      |> assign_new(:checked, fn -> get_checkbox_value(assigns) end)
+      |> assign_new(:disabled, fn -> false end)
+      |> assign_new(:required, fn -> false end)
+      |> assign_new(:tooltip, fn -> nil end)
+
+    ~H"""
+    <div
+      id={"toggle-container-#{@id}"}
+      class={["flex flex-col gap-1", @class]}
+      {if @tooltip, do: ["phx-hook": "Tooltip", "aria-label": @tooltip], else: []}
+    >
+      <div
+        phx-hook="Toggle"
+        id={"toggle-control-#{@id}"}
+        class="flex items-center gap-3"
+        {if @on_click, do: ["phx-click": JS.push(@on_click,
+          value: %{
+            _target: @name,
+            "#{@name}": !@checked,
+            value_key: to_string(@value_key)
+          }
+        )], else: []}
+      >
+        <div class="relative inline-flex items-center">
+          <input type="hidden" name={@name} value="false" />
+          <input
+            type="checkbox"
+            id={@id}
+            name={@name}
+            value="true"
+            class="sr-only peer"
+            checked={@checked}
+            disabled={@disabled}
+            required={@required}
+            {@rest}
+          />
+
+          <div
+            data-toggle
```

Comment:
What are these `data-toggle`, `data-handle` etc for? I can't quite find what is referencing them?

Reply:
Same as my previous comment. Leftovers of first design.